students/scotchwsplenda/Lesson_5/MailRoom_3.py

```diff
@@ -0,0 +1,124 @@
+
+from operator import itemgetter
+import sys
+
+
+# donors
+donators = {"Gordian": [30.0, 45.0], "Tiberius": [60.0],
+            "Maximus": [65.0, 12.0], "Tacitus": [33.0, 22.0, 25.00],
+            "Commodus": [43.0, 11.0]}
+
+
+# Opening menu
+def main():
+    print('''
+      Welcome to the MAILROOM main menu
+      Please choose from below options:
+      1 - Initiate Grovel Sequence
+      2 - Data Metrics 3000
+      3 - Quit
+      4 - Send thank you notes to all donors
+              ''')
+    while True:
+        try:
+            choice = int(input('Indicate your choice: '))
+            swit_dic = {1: send_note, 2: data_metrics,
+                        3: exit_program, 4: mass_mail}
+            swit_dic[choice]()
```

Comment:
you should be catching a KeyError here; what if they enter '5'?

you also don't need the ValueError at all - just catch KeyError.